mlos_bench/mlos_bench/optimizers/manual_optimizer.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""Optimizer for mlos_bench that proposes an explicit sequence of configuration."""
+
+import logging
+from typing import Dict, List, Optional
+
+from mlos_bench.optimizers.mock_optimizer import MockOptimizer
+from mlos_bench.services.base_service import Service
+from mlos_bench.tunables.tunable import TunableValue
+from mlos_bench.tunables.tunable_groups import TunableGroups
+
+_LOG = logging.getLogger(__name__)
+
+
+class ManualOptimizer(MockOptimizer):
+    """Optimizer that proposes an explicit sequence of tunable values."""
+
+    def __init__(
+        self,
+        tunables: TunableGroups,
+        config: dict,
+        global_config: Optional[dict] = None,
+        service: Optional[Service] = None,
+    ):
+        super().__init__(tunables, config, global_config, service)
```

Comment:
Add max_cycles vs max_suggestions computation here.

Reply:
Added the max_cycles computation!